src/ol_infrastructure/infrastructure/aws/opensearch/__main__.py

```diff
@@ -111,27 +189,25 @@
         ]
     ),
 )
-
 opensearch_node = consul.Node(
     "aws-opensearch-consul-node",
     address=search_domain.endpoint,
     opts=pulumi.ResourceOptions(provider=consul_provider),
 )
-
 opensearch_service = consul.Service(
     "aws-opensearch-consul-service",
     node=opensearch_node.name,
-    name="elasticsearch",
+    name="opensearch",
```

Comment:
I should make this a configuration setting `opensearch:consul_service_name` and then update the old stacks to set it to `elasticsearch`

Reply:
I think that's a good practice in general, that way it allows for multiple clusters to co-exist in the same Consul datacenter without conflicting.